### PR TITLE
Update links in snapshot timestamp sync playbook

### LINKED ISSUE #889: snapshot-timestamp-sync.md needs updated links
[snapshot-timestamp-sync.md](https://github.com/sigstore/root-signing/blob/main/playbooks/snapshot-timestamp-sync.md) is out of date.

### PR DESCRIPTION
**Pull request by lkatalin**
#### Summary
Just updating `snapshot-timestamp-sync.md` as I go through it and find dead links, etc.
Fixes #889 

#### Release Note
NONE

#### Documentation
N/A

### CODE REVIEW

**Review by Hayden-IO — DISMISSED** [review]

**Comment by Hayden-IO** [comment]
@lkatalin is this good to move out of draft? Everything else should be up to date.

**Comment by lkatalin** [comment]
@haydentherapper I also get a 404 on the link https://github.com/sigstore/public-good-instance/blob/main/.github/workflows/reusable-prober.yml , but not sure where it is supposed to go instead.

**Comment by Hayden-IO** [comment]
Ah, that's now https://github.com/sigstore/sigstore-probers/blob/main/.github/workflows/reusable-prober.yml

**Comment by lkatalin** [comment]
Thanks! I've updated it, but I think the line numbers may be wrong on the reusable prober link. I'll have to dig into it further unless you have a hint.

File: playbooks/snapshot-timestamp-sync.md [hunk 1]
```diff
@@ -11,10 +11,10 @@ with the metadata, using a Cloud KMS key to sign the metadata. There is also a w
 timestamp metadata midweek.
 
 After being approved and merged, the metadata is [synced](../.github/workflows/sync-main-to-preprod.yml)
-to a preproduction GCS bucket. Probers run against the bucket to (Repo only visible to Sigstore infrastructure developers):
+to a preproduction GCS bucket. Probers run against the bucket to:
 
-* Verify an artifact using the production TUF repository [here](https://github.com/sigstore/public-good-instance/blob/main/.github/workflows/reusable-prober.yml#L220-L249)
-* Verify the repository and the expiration of the metadata [here](https://github.com/sigstore/public-good-instance/blob/main/.github/workflows/reusable-prober.yml#L134-L156)
+* Verify an artifact using the production TUF repository [here](https://github.com/sigstore/sigstore-probers/blob/main/.github/workflows/reusable-prober.yml#L220-L249)
+* Verify the repository and the expiration of the metadata [here](https://github.com/sigstore/sigstore-probers/blob/main/.github/workflows/reusable-prober.yml#L134-L156)
```
Comment (Hayden-IO) [inline playbooks/snapshot-timestamp-sync.md]: https://github.com/sigstore/sigstore-probers/blob/318b3660fdaa8f9b6b98c872c0e7e7b11fb4938b/.github/workflows/reusable-prober.yml#L106-L170

File: playbooks/snapshot-timestamp-sync.md [hunk 2]
```diff
@@ -11,10 +11,10 @@ with the metadata, using a Cloud KMS key to sign the metadata. There is also a w
 timestamp metadata midweek.
 
 After being approved and merged, the metadata is [synced](../.github/workflows/sync-main-to-preprod.yml)
-to a preproduction GCS bucket. Probers run against the bucket to (Repo only visible to Sigstore infrastructure developers):
+to a preproduction GCS bucket. Probers run against the bucket to:
 
-* Verify an artifact using the production TUF repository [here](https://github.com/sigstore/public-good-instance/blob/main/.github/workflows/reusable-prober.yml#L220-L249)
-* Verify the repository and the expiration of the metadata [here](https://github.com/sigstore/public-good-instance/blob/main/.github/workflows/reusable-prober.yml#L134-L156)
+* Verify an artifact using the production TUF repository [here](https://github.com/sigstore/sigstore-probers/blob/main/.github/workflows/reusable-prober.yml#L220-L249)
```
Comment (Hayden-IO) [inline playbooks/snapshot-timestamp-sync.md]: https://github.com/sigstore/sigstore-probers/blob/318b3660fdaa8f9b6b98c872c0e7e7b11fb4938b/.github/workflows/reusable-prober.yml#L245-L304

**Review by Hayden-IO — COMMENTED** [review]
Added the line numbers

**Review by Hayden-IO — APPROVED** [review]

**Pull request merged**